src/main/resources/static/api.yaml

```diff
@@ -1561,6 +1561,103 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+  /patients/forgot-pin/generate-otp:
+    post:
+      tags:
+        - account
+      summary: Generate OTP for forgot pin flow
+      parameters:
```

Comment:
It's fine now, we should try to unify the all OTP based thing into one single API, where generate\_OTP should take the on what scope user is trying to generate OTP.

Reply:
Yes, same thing we were thinking, We will create a tech debt card for that.